vmaas_sync/repo_based.go

```diff
@@ -87,41 +86,38 @@ func getRepoBasedInventoryIDs(repos []string) ([]inventoryAID, error) {
 }
 
 //nolint unused
-func getUpdatedRepos(modifiedSince *time.Time) ([]string, error) {
+func getUpdatedRepos(modifiedSince *time.Time, thirdParty bool) ([]string, error) {
 	page := float32(1)
 	var reposArr []string
 	for {
 		reposReq := vmaas.ReposRequest{
-			Page:           page,
+			Page:           vmaas.PtrFloat32(page),
 			RepositoryList: []string{".*"},
-			PageSize:       float32(advisoryPageSize),
+			PageSize:       vmaas.PtrFloat32(float32(advisoryPageSize)),
+			ThirdParty:	    vmaas.PtrBool(thirdParty),
 		}
 
 		if modifiedSince != nil {
-			reposReq.ModifiedSince = modifiedSince.Format(time.RFC3339)
+			reposReq.SetModifiedSince(*modifiedSince)
```

Comment:
Are you sure the formatting can be removed?

Reply:
Yes, because reposReq.ModifiedSince is now time.Time not string.